### PR TITLE
I had slightly wrong formulas and the wrong ipc tables for some parts

### PR DESCRIPTION
**Pull request by poeschlr**
Issue on the footprint libs: https://github.com/KiCad/kicad-footprints/issues/466

Footprints have not yet been merged. (Will do it tomorrow as i am a bit too tired right now. This is such that the reporters can review my fix)

### CODE REVIEW

File: scripts/SMD_chip_package_rlc-etc/ipc_smd_two_terminal_chip.yaml [hunk 1]
```diff
@@ -63,24 +63,24 @@ ipc_spec_smaller_0402:
...
+# Side (JS) |  -0.05  |  0.00     |  0.05
 # Courtyard |  0.1    |  0.25     |  0.5
 
 ipc_spec_larger_or_equal_0603:
     least:
         toe: 0.15
-        heel: -0.3
-        side: 0.01
+        heel: -0.05
+        -side: 0.05
```
Comment (sethhillbrand) [inline scripts/SMD_chip_package_rlc-etc/ipc_smd_two_terminal_chip.yaml]: I think this '-' is a typo

File: scripts/SMD_chip_package_rlc-etc/SMD_chip_package_rlc-etc.py [hunk 2]
```diff
@@ -60,17 +60,22 @@ def calcPadDetails(self, device_params, ipc_data, ipc_round_base, footprint_grou
...
+            Tmax = (device_params['body_length_min'] - device_params['terminator_spacing_min'])/2
+            Ttol = (Tmax-Tmin)/2
+            Stol_RMS = math.sqrt(Ltol**2+2*(Ttol**2))
+            Smax_RMS = device_params['terminator_spacing_min'] + Stol_RMS
+            Gmin = Smax_RMS - 2*ipc_data['heel'] - math.sqrt(Stol_RMS**2 + F**2 + P**2)
+
         else:
-            terminal_tolerance = device_params['terminal_length_max'] - device_params['terminal_length_min']
-            spacing_tolerance = math.sqrt(length_tolerance**2+terminal_tolerance**2)
+            Ttol = (device_params['terminal_length_max'] - device_params['terminal_length_min'])/2
```
Comment (sethhillbrand) [inline scripts/SMD_chip_package_rlc-etc/SMD_chip_package_rlc-etc.py]: I suspect that Ttol should not be divided by 2 here. (or above on line 65)  It doesn't look like you are measuring from the center of the component.  These are absolute lengths

Comment (poeschlr) [reply scripts/SMD_chip_package_rlc-etc/SMD_chip_package_rlc-etc.py]: Yes it needs to be devided by two. The reason is that the tolerance is relative to the nominal size which is assumed to be between max and min.

Comment (poeschlr) [reply scripts/SMD_chip_package_rlc-etc/SMD_chip_package_rlc-etc.py]: Ok i re read the relevant parts of the standard. I think you might be right.

**Comment by poeschlr** [comment]
I fixed all the mentioned errors.

The footprints are now added in PR https://github.com/KiCad/kicad-footprints/pull/469

**Comment by poeschlr** [comment]
@pointhi the new footprints have been merged -> this is now also ready to be merged.

**Comment by pointhi** [comment]
thanks, I will merge

**Pull request merged**